### ISSUE TITLE
Frontpage redesgin

### ISSUE THREAD

**Issue by c055** [bitshares/bitshares.github.io-old#95]

**Comment by jonnybitcoin** [comment]
when is this being released?

**Comment by c055** [comment]
i will contiue working on this by mid of this week .. i'm offline currently due a bereavement in my family .. sry for circumstances ...

**Comment by c055** [comment]
![bts frontpage redesign ideas](https://cloud.githubusercontent.com/assets/6280810/13133152/14099104-d5f9-11e5-8d82-30d67eb88db9.png)

mostly blind text yet .. but in this direction ...
on Dex info section i've use ur content @jonnybitcoin from dex site .. hope this was ok for now ..

**Comment by svk31** [comment]
I like the screenshot and the graphical design! Please keep the text simple and understandable though, no fancy buzzwords and tech jargon.

**Comment by c055** [comment]
ok but anyhow we have to get words in...
- smart contract platform
- distributed exchange
- governance

**Comment by jonnybitcoin** [comment]
There may be merit in coding the website in react.js It's fast and beautiful and can display realtime data from the blockchain in the future. 
Eg the video background I have at thedex.org could actually be the real live data and graphs not just an out of date screen recording.

**Comment by jonnybitcoin** [comment]
Mixed market messages are a bad idea. I thought graphene was just a code name for BTS2 but for some it sticks.
The backend is named bitshares-2 on github and the frontend is bitshares-2-ui and these use the bitshares blockchain.

Why confuse newbies with the word graphene. Unless we want to rebrand bitshares graphene lets kill it off now.

theres enough confusion with openledger being the official web client.

**Comment by c055** [comment]
yep agree dan mentioned it also .. let me go into 2nd iteration ..

**Comment by c055** [comment]
site is hosted via github pages and jekyll is easy for all devs to work with ..

**Comment by c055** [comment]
i'm thinking we shouldn'T switch the framework ... otherwise the project will consume much more time to get done .. @valzav thoughts? Are u happy with jekyll and working in this manner on github?
